services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/services/ReportService.java

```diff
@@ -59,6 +61,8 @@ public class ReportService {
     private final MapBroadcastRateEventRepository mapBroadcastRateEventRepo;
     private final ReportRepository reportRepo;
 
+    int maximumResponseSize;
+
```

Comment:
chore(non-blocking): make this private and final like the rest of the class properties to stay consistent and properly encapsulate. 
```suggestion
    private final int maximumResponseSize;

```

Reply:
updated